app/model/hf_runtime.py

```
@@ -0,0 +1,253 @@
+                        # Normalize logits to numpy array
+                        if hasattr(logits, "detach"):
+                            arr = logits.detach().cpu().numpy()
+                        elif hasattr(logits, "numpy"):
+                            arr = logits.numpy()
+                        else:
+                            import numpy as _np
+                            arr = _np.asarray(logits)
+                        return arr
+                except Exception as e:
+                    # raise to be caught by outer coroutine
+                    raise
```

Ruff (0.13.1)

Remove exception handler; error is immediately re-raised

(TRY203)